frontend/src/metabase/admin/performance/components/StrategyForm.tsx

```diff
@@ -117,11 +122,38 @@ export const StrategyForm = ({
         shouldShowName={shouldShowName}
         buttonLabels={buttonLabels}
         isInSidebar={isInSidebar}
+        strategyType={initialValues.type}
       />
     </FormProvider>
   );
 };
 
+/** Don't count the addition/deletion of a default value as a reason to consider the form dirty */
+const isFormDirty = (values: CacheStrategy, initialValues: CacheStrategy) => {
+  const fieldNames = [...Object.keys(values), ...Object.keys(initialValues)];
```

Comment:
```suggestion
  const fieldNames = _.uniq([...Object.keys(values), ...Object.keys(initialValues)]);
```
is this needed to avoid duplicate keys?